ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -539,17 +538,12 @@ def get_disk_safety(logger):
                 "lost_disks": None
             }
 
-        abms = set(service.name.replace('arakoon-', '') for service in ServiceHelper.get_services()
-                   if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
+        abm_services = set(service for service in ServiceHelper.get_services() if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
```

Comment:
These are not ABM services, but normal services of the type ALBA_MGR, so i'd rename this to ```services``` or collect the respective abm_service property of each service found like this:
```
abm_services = set(service.abm_service for service in ServiceHelper.get_services() if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
```